src/grain-stdlib/lists.grlib

```diff
@@ -1,64 +1,64 @@
 # Standard library for list functionality
 
-data a List =
-  | empty
-  | cons(a, a List);
+data List<a> =
+  | Empty
+  | Cons(a, List<a>);
 
 let rec length = (lst) => {
   match (lst) {
-    | empty => 0
-    | cons(hd, tl : a List) => 1 + length(tl)
+    | Empty => 0
+    | Cons(hd, tl) => 1 + length(tl)
```

Comment:
Why drop the explicit type annotation here? Just for aesthetics? Just curious.

Reply:
Honestly, I thought I added it in when I was testing. It was breaking during parsing (after I updated the grammar) and I just removed it. I can add it back if you care. This goes to the larger point, though, of philosophy. I was leaning towards using type annotations as a debugging tool rather than for code, but obviously adding them in if it makes the code more readable. Here, I feel like it's clear that it's a list. It'll be even more clear once we have macros/syntax support for lists.